book/emails.rst

```diff
@@ -0,0 +1,340 @@
+.. index::
+    :single: E-Mails
+
+Working with E-Mails
+====================
+
+Communicating with customers is an important part of every business. In the Oro Platform, an e-mail
+address is represented by the :class:`Oro\\Bundle\\EmailBundle\\Entity\\EmailAddress` activity.
```

Comment:
There is still some confusion:
As was mentioned before - an e-mail address is represented by EmailAddress class, and email instance is represented by Email class. But only Email is **activity**, EmailAddress is not.